lib/paper_trail/has_paper_trail.rb

```diff
@@ -46,6 +47,12 @@ module ClassMethods
       #   the instance was reified from. Default is `:version`.
       # - :save_changes - Whether or not to save changes to the object_changes
       #   column if it exists. Default is true
+      # - :join_tables - If the model has a has_and_belongs_to_many relation
+      #   with an unpapertrailed model, passing the name of the association to
+      #   the join_tables option will paper trail the join table but not save
+      #   the other model, allowing reification of the association but with the
+      #   other models latest state (if the other model is paper trailed, this
+      #   option does nothing)
```

Comment:
Hi, if both models are paper trailed then the join table is also paper trailed, allowing a complete history of the models and associations. If only one model is paper trailed, and the other is not, then by default nothing is saved in relation to both models. However, by passing the `join_tables` option, the association changes are saved. You would only ever need `join_tables` when you want to paper trail an association but not the associated model. If you take a look at `save_associations_has_and_belongs_to_many` you can see the logic behind the saving.